website/blog/2020-06-27-lithium.md

```diff
@@ -0,0 +1,503 @@
+---
+author: Tomasz Godzik
+title: Metals v0.9.1 - Lithium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We are happy to announce the release of Metals v0.9.1, which concentrated on
+fixing issues with recently released Scala 3 support, Ammonite support and a
+couple of new features.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">261</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">96</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">16</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">12</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">3</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/24?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Ammonite support
+- new project provider
+- worksheet dependency support
+- Scala 2.13.3, 0.24.0, 0.25.0-RC1 support
+
+## Ammonite support
+
+Thanks to a great work by [alexarchambault](https://github.com/alexarchambault)
+we know support [Ammonite](https://ammonite.io) script files. Ammonite enables
+to create scripts that you can run easily from the command line, without the
+overhead of setting build tool while still allowing you to download and use
+dependencies.
+
+The support includes most of the LSP features such as diagnostics, completions,
+references, go to definition, but does not include running or debugging the
+scripts. This new feature works by starting an additional Ammonite BSP server
+that compiles the scripts and provides additional information about them such as
+scala version or information about the added dependencies. You can change the
+default JVM parameters for the server such as `-Xmx`, which will limit the used
+memory in Metals settings.
+
+Each of the scripts needs to be first imported into Ammonite BSP server, which
+can be done manually by clicking on the pop up whenever new script is opened or
+selecting automatic import at the start, which will import any open scripts.
```

Comment:
Ah maybe what I mentioned about the start and stop commands could be mentioned here?

Reply:
I think it's more natural to mention it after the mentioning the BSP server.

Reply:
Sure, sounds good.